### DISCUSSION TITLE
Why I have all Word open, I want all in one windows pined in taskbar ?

### DISCUSSION THREAD

**Discussion by sypqys** [ramensoftware/windhawk-mods#427] [Q&A] [answered]
Hi !

How I have to do for this ?
![explorer_FHjnKyHnjx](https://github.com/ramensoftware/windhawk-mods/assets/27967268/f066681a-78fc-4afc-988e-3a60d0df4095)

Thanks

**Comment by m417z** [comment]
Do you want to have just a single Word icon? You have the "Disable grouping on the taskbar" mod installed, excluding Word should help so the icons will become grouped and combined.

**Comment by sypqys** [comment]
Yes I want single icon... But I don't know why he dont work... I desactivate "disable grouping on the taskbar" and is the same...

**Reply by m417z** [reply, marked as answer]
After disabling the mod, you need to close and re-open all relevant windows to apply the new grouping rules.

**Reply by sypqys** [reply]
Thank you very much, it is all OK !
